generators/spring-controller/index.js

```diff
@@ -116,6 +116,20 @@ module.exports = class extends BaseBlueprintGenerator {
     return this._prompting();
   }
 
+  // Public API method used by the getter and also by Blueprints
+  _loading() {
+    return {
+      loadSharedConfig() {
+        this.loadDerivedServerConfig();
```

Comment:
`loadDerivedServerConfig` requires `loadServerConfig`
```suggestion
        this.loadServerConfig();
        this.loadDerivedServerConfig();
```